api_tests/meetings/views/test_meetings_submissions_detail.py

```diff
@@ -56,7 +56,15 @@ def file(self, user, meeting_one_submission):
         return file
 
     def mock_download(self, project, file, download_count):
-        return PageCounter.objects.create(_id='download:{}:{}'.format(project._id, file._id), total=download_count)
+        pc, _ = PageCounter.objects.get_or_create(
+            _id='download:{}:{}'.format(project._id, file._id),
```

Comment:
So we still need the existing `_id`? I forget where we are in the process or if we are planning on removing those at some point.

Reply:
There will be a third follow-up for this one! https://openscience.atlassian.net/browse/ENG-573.  Back when Steve was reviewing it, removing the _id was originally part of this ticket, but was recommended to split it off into a third final piece.  That ticket will convert the _id to field to just a long _id.